calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,123 @@
+| `lidartag_max_convergence_translation`        | `double`              | `0.05`                           | Threshold in meters for the translation component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                                     |
+| `lidartag_max_convergence_translation_dot`    | `double`              | `0.03`                           | Threshold in meters per second for the velocity component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                             |
+| `lidartag_max_convergence_rotation`           | `double`              | `3.0`                            | Threshold in degrees for the rotation component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                                       |
+| `lidartag_max_convergence_rotation_dot`       | `double`              | `2.5`                            | Threshold in degrees per second for the angular velocity component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                    |
+| `lidartag_new_hypothesis_translation`         | `double`              | `0.1`                            | Translation threshold in meters for generating a new hypothesis in lidartag tracking.                                                                                                                                                                                   |
+| `lidartag_new_hypothesis_rotation`            | `double`              | `15.0`                           | Rotation threshold in degree for generating a new hypothesis in lidartag tracking.                                                                                                                                                                                      |
+| `lidartag_measurement_noise_translation`      | `double`              | `0.05`                           | The square of this value in meters is part of the input for the measurement noise covariance matrix (R) in the Kalman filter.                                                                                                                                           |
```

Comment:
We already talked about not using 
`The square of this value in meter`

At that time you should have applied everywhere it applied, not only to the instances I pointed out. Same goes for the units

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/be88f117fda0069e3d31d0a75dfe7546df9c7baf

Reply:
meters per second (meters/second)
degrees per second (degrees/second)

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/1d660e71e48d47fd120f42b8a50fa996643f26aa